### PR TITLE
Enhancement/update dependencies

### LINKED ISSUE #120: Warnings on node@6
`(node:1815) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.`

Looks like one of the dependencies is using `graceful-fs` and needs updated.

### PR DESCRIPTION
**Pull request by harlanj**
- bump `fstream-ignore`
- bump `update-notifier`

Removes `node@v6` installation warning
Closes #120

### CODE REVIEW

File: lib/modulus.js [hunk 1]
```diff
@@ -80,7 +81,7 @@ modulus.runCommand = function (command, args, authRequired) {
   modulus.io.print('Welcome to ' + 'Modulus'.magenta);
 
   var diff = require('update-notifier')({
-    packagePath: path.resolve(__dirname, '..', 'package.json')
```
Comment (jackboberg) [inline lib/modulus.js]: replaced `path` with `json`...?

Comment (harlanj) [inline lib/modulus.js]: Via `update-notifier`'s [documentation](https://www.npmjs.com/package/update-notifier).

**Comment by jackboberg** [comment]
👍 assuming your tests all worked

**Comment by harlanj** [comment]
Test cases

- [ ] Deploy a project with a `.modulusignore` file (tests `fstream-ignore`)
- [ ] **Run the CLI with older version of `package.json` (tests `update-notifier`)

** I'm not sure if the results that `update-notifier` gets are cached. It sometimes notified when the version of the CLI was outdated. I tested this also by `npm i -g modulus@5.0.1`, then ran the `modulus` command. I was not prompted that I was using an outdated version.

**Comment by musgravejw** [comment]
👍

**Pull request merged**